pkg/build/builder/daemonless.go

```diff
@@ -555,6 +561,9 @@ func GetDaemonlessClient(systemContext types.SystemContext, store storage.Store,
 		return nil, fmt.Errorf("unrecognized BUILD_ISOLATION setting %q", strings.ToLower(isolationSpec))
 	}
 
+	isolation = buildah.IsolationOCI
```

Comment:
Ok @nalind  I think we should tweak this slightly to where we supply a default of `IsolationOCI` but cans still override it vai the `BUILD_ISOLATION` env var (in case say there are performance implications for using `IsolationOCI` vs. `IsolationChroot`)

As such, the case statement above here at https://github.com/openshift/builder/blob/master/pkg/build/builder/daemonless.go#L555 should change the default case to simply pick OCI vs. returning an error

Then, after this merges.  We can craft an OCM PR where we remove the setting of `BUILD_ISOLATION` to chroot at https://github.com/openshift/openshift-controller-manager/blob/master/pkg/build/controller/strategy/util.go#L457 and https://github.com/openshift/openshift-controller-manager/blob/master/pkg/build/controller/strategy/util.go#L491

thoughts @nalind @bparees ?

Reply:
Having OCM change what it tells the builder image to use is the more technically correct way to do this, since we've been providing that particular control knob for some time.  Forcing it here was also about flushing out possible breakage for that case, but all things being equal I'd prefer to let OCM continue to make that decision.